fix csv reader bug

xixiaoyao opened the pull request:
str.split returns a **list** even though there exists only one column in text. No need to add extra dimension for that case, or the returned value will be a 2-dim list then raise errors when creating namedtuple after.

Meiyim commented:
i dont think there's  a problem with 1 column:
try this:
```python
In [1]: from collections import namedtuple

In [2]: E = namedtuple('Example', ('text_a',))

In [3]: args = ('foo',)

In [4]: E(*args)
Out[4]: Example(text_a='foo')
```

xixiaoyao commented:
> i dont think there's a problem with 1 column:
> try this:
> 
> ```python
> In [1]: from collections import namedtuple
> 
> In [2]: E = namedtuple('Example', ('text_a',))
> 
> In [3]: args = ('foo',)
> 
> In [4]: E(*args)
> Out[4]: Example(text_a='foo')
> ```

Noted that `['foo'],`is not equal to `['foo',]`. Check again please.

See this,
```python
>>> "text_a".split('\t'),
(['text_a'],)

>>> from collections import namedtuple
>>> a="text_a".split('\t'),
>>> namedtuple(a)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: namedtuple() missing 1 required positional argument: 'field_names'
```
